CONTRIBUTING.md

```diff
@@ -25,13 +26,76 @@ _As contributors and maintainers of this project, and in the interest of fosteri
 
 ## Getting Started
 
-Dev guides:
-
-- [Contribution Guide]
-- [MacOS Dev Guide]
-- [Windows Dev Guide]
-
-General resources for contributors:
+### Forking Kustomize and Working Locally
+The Kustomize project uses a "Fork and Pull" workflow that is standard to GitHub. In git terms, your personal fork is referred to as the "origin" and the actual project's git repository is called "upstream". To keep your personal branch (origin) up to date with the project (upstream), it must be configured within your local working copy.
+
+### Create a fork in GitHub
+1. Visit https://github.com/kubernetes-sigs/kustomize
+2. Click the `Fork` button on the top right
+
+### Clone the repository
+```bash
+# Clone your repository fork from the previous step
+export user=<your github profile name>
+git clone --recurse-submodules git@github.com:$user/kustomize.git
```

Comment:
How about [`github username`](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) instead of `github profile name`?

And I personally think below would be enough to understand.

```suggestion
git clone --recurse-submodules git@github.com:<your github username>/kustomize.git
```

Reply:
Thanks for the suggestion. I updated this.